buffer/buffer.go

```diff
@@ -1050,3 +1053,30 @@ func (buffer *Buffer) getMaxLines() uint64 {
 
 	return result
 }
+
+func (buffer *Buffer) Save(path string) {
```

Comment:
Should the `Save()` and `Compare()` functions be moved to `buffer_test()` file as they're used only for testing?

Reply:
@nikitar020 thank you for you comment, unfortunately, since `Compare()` is used from `main_test.go`, if I move the definition to `buffer_test.go` the method can't be resolved during run of 'go test' on Linux. Haven't found how to overcome this issue.